inc/thorvg.h

```diff
@@ -990,6 +990,16 @@ class TVG_API Shape final : public Paint
      */
     Result stroke(StrokeJoin join) noexcept;
 
+
+    /**
+     * @brief Sets the stroke miterlimit.
+     *
+     * @param[in] miterlimit The miterlimit imposes a limit on the extent of the stroke join, when the @c StrokeJoin::Miter join style is set. The default value is 4.
+     *
+     * @return Result::Success when succeed, Result::NonSupport unsupported value, Result::FailedAllocation otherwise.
+     */
```

Comment:
Please add @BETA_API

(You can see how other beta apis have the tag in thorvg.h)